modular_skyrat/modules/customization/modules/clothing/under/accessories.dm

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
why this

Reply:
to get it to work, you wanted it ticked so I ticked it, made it white space, I was gonna add the tie as an accessory before I realised doing it as a tie was better

Reply:
ya, remove the file and include :)

Reply:
done